packages/ui-core/src/components/Preloader/Preloader.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import './Preloader.less';
+import cnCreate from 'utils/cn';
+
+interface IPreloaderProps {
+    /** Color themes of preloader's circles */
+    color?: 'default' | 'black' | 'white';
+    /** Sizes of preloader */
```

Comment:
```suggestion
    /** Preloader size */
```
тут же не множественное число, при использовании выбирается один вариант